### DISCUSSION TITLE
My contributions are not showing in the profile.

### DISCUSSION THREAD

**Discussion by iam-gopi** [community/community#74676] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Team,

I make contributions to some clients. but that doesn't show up in the contribution list. It's been happening to me for the last week. 
If you look at the screenshot below the contributions are not displaying. but when I got into the repos that I am working I could see my pushes.  Note the highlighted area.

![image](https://github.com/community/community/assets/18674571/ca8e9e71-5f2e-4eca-a4d4-08dfa34e7457)

Can someone say the reason?

**Comment by JaberWiki** [comment]
@iam-gopi make your private contributions public. You can check the image below.

![image](https://github.com/community/community/assets/55241455/099277f2-fec0-41db-9da2-00d203c1089b)

**Reply by iam-gopi** [reply]
As you know, private repository changes should not be visible outside. moreover, those options are already checked.  I think that's default (im not sure though)

**Comment by JaberWiki** [comment, marked as answer]
If your code is not merged in main branch it will not show in your contribution.

**Reply by lucasbrun815** [reply]
Ye, this is the reason !!

**Reply by iam-gopi** [reply]
oh yeah, all my pushes are to another branch. thanks that might be the reason :)

**Comment by karandulloo** [comment]
in  my case i dont merge the PR i have just created this public repo and i push the code to the master on the daily basis but it does not show in the contributions

**Reply by cyberdslayer** [reply]
i am having the sam issue dont know what to do

**Reply by karandulloo** [reply]
@cyberdslayer  Hey bro , i found the solution. SO basically you need to create a PR for every change. Push the changes from your local repo to the remote and then approve and merge the PR. you will be able to see the green mark then.

**Comment by XynaxDev** [comment]
hii, 
i am getting the same issue of not showing the contribution graph squares, i think this is happening because i have changed my email address and old email is tied up with old commits and now i have deleted that and have added new email so after this i'm not getting my all commits, what should i do? i have tried to add my old email in my profile but still its not reflecting that graph.
